modules/loader/src/main/resources/application.conf

```diff
@@ -6,7 +6,9 @@
       }
     },
     "schedules": {
-        "noOperation": []
+        "noOperation": [],
+        "optimizeEventsSchedule": "0 0 0 ? * *",
```

Comment:
I am a bit uneasy with this default schedules. It seems to me they are too much opinionated. Would it better to leave them empty instead ?